src/libraries/System.Private.CoreLib/src/System/Environment.Browser.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    public static partial class Environment
+    {
+        // Emscripten VFS mounts at / and is the only drive
+        public static string[] GetLogicalDrives() => new string[] { "/" };
+
+        // In the mono runtime, this maps to gethostname, which returns 'emscripten'.
+        // Returning the value here allows us to exclude more of the runtime.
+        public static string MachineName => "emscripten";
+
+        // 3.2 provides no context, so return the same value.
+        public static long WorkingSet => 0;
+
+        // 3.2 provides no context, so return the same value.
+        public static string UserName => "web_user";
```

Comment:
Can we return just empty string?

Reply:
I think it makes more sense to stick w/ this or some kind of default.  I don't think it should be an empty string.

Reply:
It's not clear this is "some kind of default" is it?

Reply:
No, it’s slightly better than emscripten 😜.  Open to suggestions.

Reply:
Suggestions: `Browser`, `.NET`, `UserName`

Reply:
Any of @marek-safar's suggestions sound reasonable to me.  Or maybe "Unknown".  I'm also fine with an empty string.